app/src/main/java/com/benavalli/arcmovies/features/home/presenter/DetailsActivityPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package com.benavalli.arcmovies.features.home.presenter
+
+import android.content.Context
+import android.util.Log
+import com.benavalli.arcmovies.api.repository.HomeRepository
+import com.benavalli.arcmovies.base.BasePresenter
+import com.benavalli.arcmovies.domain.MovieDetails
+import com.benavalli.arcmovies.domain.UpcomingMovies
+import com.benavalli.arcmovies.features.home.contract.DetailsActivityContract
+import javax.inject.Inject
+
+class DetailsActivityPresenter
+@Inject
+constructor(private val context: Context, private val repository: HomeRepository) : BasePresenter<DetailsActivityContract.View>(),
```

Comment:
seems that you can remove the context here too

Reply:
removed